tests/app/service/test_rest.py

```diff
@@ -2491,6 +2491,43 @@ def test_get_email_reply_to_addresses_with_multiple_email_addresses(client, noti
     assert not json_response[1]['updated_at']
 
 
+@freeze_time("2016-01-01 11:09:00.061258")
+def test_verify_new_service_reply_to_email_address_should_send_verification_email(
+    client, sample_user, mocker, verify_reply_to_address_email_template
```

Comment:
sample_user is unused here. (and in the other tests)

however, if you remove it then you'll need to either add `notify_db_session` as a fixture or move from `create_service()` to using `sample_service` instead (since `sample_service` has `notify_db_session` as a fixture dependency)

Reply:
good point, done :)